WcaOnRails/config/locales/en.yml

```diff
@@ -330,6 +333,9 @@ en:
         showAtAll: ""
         start_date: ""
         use_wca_registration: ""
+        competitor_limit_flag: "This is currently for information purposes only and will not affect the registration flow. We are working on adding explicit support for this to the registration flow, but it requires some other work first."
```

Comment:
How about "... and will not prevent competitors from registering for a competition whose competitor limit has been reached". I think that's a little clearer than "will not affect the registration flow".